plugins/org.eclipse.glsp.ide.editor/src/org/eclipse/glsp/ide/editor/actions/InvokeDeleteAction.java

```diff
@@ -13,15 +13,18 @@
  *
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
  ********************************************************************************/
-import { Action } from "@eclipse-glsp/client";
+package org.eclipse.glsp.ide.editor.actions;
```

Comment:
Why does this file appear as renamed from keep-alive-action.ts to InvokeDeleteAction.java?

Reply:
I have no idea but I don't think I can really influence that :sweat_smile: